cardinal/server/server.go

```diff
@@ -33,7 +35,8 @@ type Handler struct {
 	port                   string
 
 	// plugins
-	adapter shard.WriteAdapter
+	adapter         shard.WriteAdapter
+	isServerRunning atomic.Bool
```

Comment:
Is this only used to make sure the server is running before tests start?

Is it required? The `makeTestTransactionHandler` helper appears to make a health check endpoint to ensure the server is up and serving traffic before letting any tests continue.

Reply:
It's very common for to check if a daemon is "running" even outside of testing. Plenty of things you don't want to do while a daemon is running and plenty of things you do want to do Only when the main loop is cycling. 

Right now it's only used for that application but this is a useful method.

Reply:
As agreed in stand up I will be removing this.